README.md

```diff
@@ -6,6 +6,11 @@ List of GitHub profiles that have awesome customisation, that you can use for in
 
 Feel free to submit a pull request with your suggestions of awesome github profiles.
 
+[![made with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com)
```

Comment:
```suggestion
[![made with love](https://forthebadge.com/images/badges/built-with-love.svg)](https:// github.com/EddieHubCommunity)
```
Would change the link to something like this

Reply:
ok, will fix them all now.